apps/comps/components/user-agents/agents-summary.tsx

```diff
@@ -78,13 +78,13 @@ export const AgentsSummary: React.FunctionComponent<{
         )}
       >
         <span className="text-secondary-foreground uppercase">highest p&l</span>
-        <span className="text-2xl font-semibold">
-          {highest ? (
-            <BigNumberDisplay value={highest.toString()} decimals={0} />
-          ) : (
-            "N/A"
-          )}
-        </span>
+        {highest && highest !== Number.MIN_SAFE_INTEGER ? (
+          <span className="text-2xl font-semibold">
+            <BigNumberDisplay value={highest.toString()} decimals={0} /> USD
```

Comment:
Added units here.  Also, during manual tests noticed that the `decimals` prop doesn't work how I thought it would.  If `highest` is equal to `-0.1` this displays "-0.1".  I would think it would round to nearest int.